treeherder/model/migrations/0055_autoclassify_status.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0054_duplicate_failure_classifications'),
+    ]
+
+    operations = [migrations.RunSQL(
+        sql="""
+BEGIN;
```

Comment:
OK, but why? This is a pretty substantial extra burden since there is no longer a string in the migration file that you can copy and check for syntax errors / run in a SQL command line. I think that the more requirements we place on people trying to use migrations for invasive changes, the more likely they are to decide to run the SQL manually against production instead.